apps/src/openni_klt.cpp

```diff
@@ -41,9 +41,9 @@
 #include <pcl/io/pcd_io.h>
 #include <pcl/keypoints/harris_2d.h>
 #include <pcl/tracking/pyramidal_klt.h>
-#include <pcl/visualization/boost.h>
 #include <pcl/visualization/image_viewer.h>
 
+#include <boost/date_time/posix_time/posix_time.hpp>
```

Comment:
Does this work for Windows system as well?

Reply:
What do you mean? Why wouldn't it?

Reply:
I didn't know if Posix time is meant to be a front-end header (API) or back-end header (implementation). Just checked that it's a front end header :)

Reply:
I also find the header structure of the boost libraries confusing. Often, there is no clear information what header should be included to be able to use a specific function (or I just didn't find such information :laughing: ). And it is also not always clear whether a header `<boost/X.hpp>` is a convenience meta-header and it is ok to include a more specific header `<boost/X/Y.hpp>`